js/libs/form.js

```diff
@@ -64,6 +64,44 @@ function addThumbnailToCanvas(imageURI, indexCanvas, self, isFileCanvas) {
   });
 }
 
+function getDataSourceColumnValues(field) {
```

Comment:
I see the same function declared in https://github.com/Fliplet/fliplet-widget-form-builder/blob/f870c19befb4c95f4d0ac4b41f51f2c6e0d07bd3/js/libs/core.js#L14

How can we avoid redeclaring what's essentially the same logic?

Reply:
Yes, they have the same logic. Where I can declare this method as I can use it in both form.js and core.js files?